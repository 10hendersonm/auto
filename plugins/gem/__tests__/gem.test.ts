import Auto from "@auto-it/core";
import Gem from "../src";

describe("Gem Plugin", () => {
  test("should do something", async () => {});
});
