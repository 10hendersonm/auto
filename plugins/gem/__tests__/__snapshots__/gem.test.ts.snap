// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gem Plugin validateConfig validates invalid configuration 1`] = `
Array [
  "\\"gem\\"

Found unknown configuration keys: command
",
]
`;
